docs/gitrepo-add.md

```diff
@@ -128,3 +128,31 @@ stringData:
 
 Note: If you don't add it any server's public key will be trusted and added.
 
+# Troubleshooting
+
+### Fleet deployment stuck in modified state
+
+When you add a Helm chart to Fleet, some of the components are modified, and this causes the "modified" flag in the Fleet environment.
```

Comment:
This is true for kustomize and raw yaml workloads in addition to helm.

Reply:
Are you suggesting that we generalize the language?

Reply:
Yes, no need to specifically mention helm here

Reply:
Changed to

> When you add a workload to Fleet

Reply:
I think this should
> When you deploy bundles to Fleet

Which we should use specific term in Fleet

Reply:
Changed to

> When you deploy bundles to Fleet